docs/content/docs/docs/api/_index.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
These files shouldn't be in a nested `content/docs/docs` directory. All of the sections should be placed directly in `content/docs`. The file tree for `content` should look like this.
```
content
└── docs
    ├── api
    │   ├── _index.md
    │   ├── java-api-quickstart.md
    │   ├── java-api.md
    │   ├── java-custom-catalog.md
    │   ├── javadoc
    │   │   └── _index.md
    │   ├── python-api-intro.md
    │   ├── python-feature-support.md
    │   └── python-quickstart.md
    ├── asf
    │   ├── _index.md
    │   ├── donate
    │   │   └── _index.md
    │   ├── events
    │   │   └── _index.md
    │   ├── license
    │   │   └── _index.md
    │   ├── security
    │   │   └── _index.md
    │   └── sponsors
    │       └── _index.md
    ├── athena
    │   └── _index.md
    ├── community
    │   ├── _index.md
    │   ├── blogs.md
    │   ├── contributing.md
    │   ├── join.md
    │   └── talks.md
    ├── dremio
    │   └── _index.md
    ├── emr
    │   └── _index.md
    ├── flink
    │   ├── _index.md
    │   ├── flink-connector.md
    │   └── flink-getting-started.md
    ├── format
    │   ├── _index.md
    │   ├── spec.md
    │   ├── terms.md
    │   └── view-spec.md
    ├── hive
    │   └── _index.md
    ├── integrations
    │   ├── _index.md
    │   ├── aws.md
    │   ├── jdbc.md
    │   └── nessie.md
    ├── prestodb
    │   └── _index.md
    ├── project
    │   ├── _index.md
    │   ├── benchmarks.md
    │   ├── how-to-release.md
    │   ├── how-to-verify-a-release.md
    │   ├── multi-engine-support.md
    │   ├── roadmap.md
    │   ├── security.md
    │   └── trademarks.md
    ├── releases
    │   ├── 0.12.1
    │   │   └── _index.md
    │   ├── 0.13.0
    │   │   └── _index.md
    │   ├── 0.13.1
    │   │   └── _index.md
    │   ├── 0.13.2
    │   │   └── _index.md
    │   ├── _index.md
    │   ├── latest
    │   │   └── _index.md
    │   └── release-notes.md
    ├── spark
    │   ├── _index.md
    │   ├── spark-configuration.md
    │   ├── spark-ddl.md
    │   ├── spark-getting-started.md
    │   ├── spark-procedures.md
    │   ├── spark-queries.md
    │   ├── spark-structured-streaming.md
    │   └── spark-writes.md
    ├── starrocks
    │   └── _index.md
    ├── tables
    │   ├── _index.md
    │   ├── configuration.md
    │   ├── evolution.md
    │   ├── maintenance.md
    │   ├── partitioning.md
    │   ├── performance.md
    │   ├── reliability.md
    │   └── schemas.md
    └── trino
        └── _index.md
```

Reply:
should be fixed now